lite/examples/pose_estimation/android/app/src/main/java/org/tensorflow/lite/examples/poseestimation/ml/MoveNet.kt

```diff
@@ -52,7 +52,7 @@ class MoveNet(private val interpreter: Interpreter) : PoseDetector {
                 Device.CPU -> {
                 }
                 Device.GPU -> {
-                    // Movenet TFLite models don't work with GPUDelegate yet
```

Comment:
need to find a fix for this, movenet losing it's essence of being fast for android devices.